cobbler/tftpgen.py

```diff
@@ -864,10 +864,14 @@ def build_kernel_options(self, system, profile, distro, image, arch: str, autoin
                     autoinstall_path = "http://%s/cblr/svc/op/autoinstall/profile/%s" \
                                        % (httpserveraddress, profile.name)
 
-            if distro.breed is None or distro.breed == "redhat":
+            if distro.breed is None or distro.breed == "redhat" or distro.breed == "redhatlegacy":
```

Comment:
Maybe do it this way?
```suggestion
            if distro.breed in [None, "redhat", "redhatlegacy"]:
```

Reply:
yes, that is more readable